packages/notebook/src/widget.ts

```diff
@@ -193,16 +193,19 @@ export class StaticNotebook extends WindowedList {
    */
   constructor(options: StaticNotebook.IOptions) {
     const cells = new Array<Cell>();
+    const windowingActive =
+      (options.notebookConfig?.windowingMode ??
+        StaticNotebook.defaultNotebookConfig.windowingMode) === 'full';
```

Comment:
When windowing gets switched of the scrollbar stays on, but the button hides. It seems like this needs to be a callback or a signal?